README.md

```diff
@@ -1,3 +1,10 @@
+## Overview
+
+This repository consists from a solution which contains four main packages
```

Comment:
typo: consists from -> consists of (I think)

Reply:
"This repository contains one solution which ..." might be better.